streams-contrib/streams-persist-elasticsearch/pom.xml

```diff
@@ -53,6 +53,27 @@
             <type>test-jar</type>
         </dependency>
         <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.9.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.streams</groupId>
+            <artifactId>streams-runtime-threaded</artifactId>
+            <version>0.2.4-W2O-SNAPSHOT</version>
```

Comment:
Why does the elasticsearch module depend on the threaded runtime? Also, we should be managing the version in the parent pom's dependency management section